react/features/screenshot-capture/createImageBitmap.js

```diff
@@ -0,0 +1,25 @@
+/*
+* Safari polyfill for createImageBitmap
+* https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/createImageBitmap
+*
+* Support source image types: Canvas.
+*/
+if (!('createImageBitmap' in window)) {
+    window.createImageBitmap = async function(data) {
```

Comment:
Let's not polyfill `window`. Just have a helper that calls the original function where available and uses your fallback when not.

Reply:
I can't do that because I'm not using it, but the ImageCapture polyfill. That polyfill uses createImageBitmap, which is not implemented in Safari and that's why I added this polyfill